src/iosource/pcap/Source.cc

```diff
@@ -247,6 +247,9 @@ bool PcapSource::ExtractNextPacket(Packet* pkt)
 	++stats.received;
 	stats.bytes_received += header->len;
 
+	header->len = 0;
+	header->caplen = 0;
```

Comment:
Suggest to add a comment here what's going on / why we do this.

Reply:
Also, a question: will setting these to zero then trigger the `empty_pcap_header` that I see a little bit above this code? And if so, wouldn't that be very noisy because it sounds like Myricom ends up doing this in a pretty standard case (i.e., not having a packet ready).

Reply:
(Btw, nevermind my earlier comment on not understanding it in case anybody sees that still, I got that part now and deleted the comment).